dist/ansible/scripts/ovn-setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+  
+# Create a configmap of the ovn configuration information
+# This script runs on the ovn-master node on the cluster
+# This is run as part of installation after the cluster is
+# up and before the ovn daemonsets are created.
+
+health=$(oc get --raw /healthz/ready)
+if [ $health != 'ok' ]
+then
+  echo " cluster not ready"
+  exit 1
+fi
+
+svcacct=$(oc get serviceaccount | grep ovn)
+if [ $? -ne 0 ]
+then
+  oc create serviceaccount ovn
+  oc adm policy add-cluster-role-to-user cluster-admin -z ovn
+  oc adm policy add-scc-to-user anyuid -z ovn
+fi
+
+# get api server
+api=$(oc get po -n kube-system -o wide | gawk '/master-api/{ print $7 }')
+apiserver=https://${api}:8443
+
+# get the API token.
+token=$(oc sa get-token ovn)
```

Comment:
As above we should just read the token via the filesystem internally in ovn-kube itself rather than putting it into the config file.  That will take a small code change in ovnkube.

Reply:
OK, do we need the token change that Rajat is doing?